move on

JavierGianneo opened the pull request:
Estan incluidos los cambios de Matias, se agregaron los modelos al a…dmin para llenarse de campos, se crearon dos nuevos forms

JavierGianneo committed:
Estan incluidos los cambios de Mathias, se agregaron los modelos al admin para llenarse de campos, se crearon dos nuevos forms

matimutter committed:
Delete Clickrentacar.iml

matimutter committed:
Delete profiles_settings.xml

matimutter committed:
Delete misc.xml

matimutter committed:
Delete modules.xml

matimutter committed:
Delete vcs.xml

matimutter committed:
Update cuentaUsuario.py

matimutter committed:
Update vehiculo.py

matimutter commented on cracAPP/views.py:
```diff
@@ -1,13 +1,20 @@
+
+def login(request):
+    login_form = Login()
+    render(request, "login.html", login_form)
```
esta funcion vamos a moverla a cracAPP\views\view_userLogin.py

matimutter commented on cracAPP/views.py:
```diff
@@ -1,13 +1,20 @@
+
+def createUser(request):
+    create_form = CreateUser()
+    render(request, "create_user.html", create_form)
```
y esta la movemos a cracAPP\views\view_userRegister.py

matimutter replied:
**DESPUES DE MOVER LAS FUNCIONES BORRAR ESTE ARCHIVO !**

matimutter commented on cracAPP/views.py:
```diff
@@ -1,13 +1,20 @@
 from django.shortcuts import render
 from django.http import HttpResponse
-
+from .forms.loginVista import Login
```
Esto esta bien, hay que ver como importar .froms desde dentro de la carpeta views.
Este archivo hay que borrarlo menos lo que comenté que va en otro lado.
Si dejas el archivo views.py hace conflicto con las views que estan en Clickrentacar\cracAPP\views\<views>.py
no puede quedar aunque no tenga nada dentro, hay que borrarlo...

matimutter commented on cracAPP/admin.py:
```diff
@@ -1,3 +1,29 @@
+admin.site.register(SolicitudAlquiler)
+admin.site.register(SolicitudRegistro)
+admin.site.register(Vehiculo)
+admin.site.register(VehiculoPapeles)
```
esto esta muy muy bien

matimutter reviewed (approved):
Todo ok, mover funciones de views.py a los archivos de la carpeta views/.
Después borrar views.py porque hace conflicto con la config.
Temporalmente mientras esté este archivo (cracAPP/views.py)  el proyecto no va a funcionar.

matimutter committed:
update templates

matimutter committed:
udpate models hay cosas para corregir

JavierGianneo committed:
Merge branch 'master' into javier

# Conflicts:
#	cracAPP/forms/createUser.py
#	cracAPP/models/cuenta.py
#	cracAPP/models/cuentaAdministrador.py
#	cracAPP/models/cuentaUsuario.py
#	cracAPP/models/vehiculo.py
#	cracAPP/views.py
#	db.sqlite3

matimutter committed:
update models

matimutter committed:
update models

En el index hise una prueba para mostrar un objeto guardado en memoria, resta guardarlo en la DB

matimutter committed:
updating models...

JavierGianneo committed:
Merge branch 'Matias' into javier

JavierGianneo committed:
Reestructuracion del proyecto en apps (todavia no funciona)

JavierGianneo committed:
Ahora funciona

matimutter committed:
UPDATE: mvc, models DELETE: pycache, migrations, db

matimutter committed:
make migrations, db

matimutter committed:
add admin

JavierGianneo committed:
Matias nueva mvc (#12)

* UPDATE: mvc, models DELETE: pycache, migrations, db

* make migrations, db

* add admin

Co-authored-by: Matias Mutter <matimutter@gmail.com>

matimutter committed:
pass en error en views index y categorias

matimutter committed:
update error view

matimutter committed:
update views TODO FUNCIONANDO

JavierGianneo committed:
Merge remote-tracking branch 'origin/matias_nueva_mvc' into javier

# Conflicts:
#	alquiler/migrations/0001_initial.py
#	cracAPP/views/view_index.py
#	db.sqlite3
#	usuario/migrations/0001_initial.py
#	vehiculo/migrations/0001_initial.py
